.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "printWidth": 80,
```

Comment:
Why is this necessary? Looks like this is the default.

Reply:
Slipped too, but it didn't seem to be my default, files were being saved funny so I manually added it there